packages/loot-core/src/server/budget/goaltemplates.js

```diff
@@ -156,12 +162,8 @@ async function applyCategoryTemplate(category, template_lines, month, force) {
           num_months = differenceInCalendarMonths(target_month, current_month);
         }
         if (num_months < 0) {
-          console.log(
-            `${category.name}: ${`${template.month} is in the past:`} ${
-              template.line
-            }`,
-          );
-          return null;
+          errors.push(`${category.name}: ${`${template.month} is in the past.`}`);
+          return [null,errors];
```

Comment:
This was incorrect in the original code — the `filter` callback should return `true`/`false`, not `null` or `template`.

Reply:
I think this is done???